LukasKaselionis/laravel-blog
# Article category relationship

Article categories many to many relation, updated article create and update forms by attach categories.

How to test:

Run php artisan migrate
Add some categories on /admin/Categories.
Try to create article and checked no categories or one category or few categories.
Try to edit article and change caegories or check no categories.
Try to delete article an check article_category table on must not to rows with delete article_id.
